pkg/network/protocols/postgres/telemetry.go

```diff
@@ -88,16 +88,20 @@ func (t *Telemetry) getBucketIndex(querySize int) int {
 func (t *Telemetry) Count(tx *ebpf.EbpfEvent, eventWrapper *EventWrapper) {
 	querySize := int(tx.Tx.Original_query_size)
 
-	bucketIndex := t.getBucketIndex(querySize)
-	if bucketIndex >= 0 && bucketIndex < len(t.queryLengthBuckets) {
-		t.queryLengthBuckets[bucketIndex].Add(1)
-	}
-
+	fulfilled := false
 	if eventWrapper.Operation() == UnknownOP {
 		t.failedOperationExtraction.Add(1)
+	} else {
+		fulfilled = true
 	}
 	if eventWrapper.TableName() == "UNKNOWN" {
 		t.failedTableNameExtraction.Add(1)
+	} else {
+		fulfilled = true
```

Comment:
As I see it, currently, `fulfilled` can be true in both cases. I think having the ability to understand what failed could give us an advantage. For example, if we were unable to extract the operation but were able to extract the table name, we wouldn't be able to identify those cases.